docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:jammy-20220531
```

Comment:
is this used anywhere in the pipelines, doesn't seems to be?

Reply:
Now it's used, thanks